metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/_config.py

```diff
@@ -41,6 +41,9 @@ class DatahubLineageConfig(ConfigModel):
     # The Airflow plugin behaves as if it were set to True.
     graceful_exceptions: bool = True
 
+    # Override the external urls of datajob.
+    override_datajob_url: str = None
+
     def make_emitter_hook(self) -> "DatahubGenericHook":
```

Comment:
configs also need to be added to `get_lineage_config` below, and also added to the docs